Update Project Profile -  Added Brandon Hu

MohamedTwahir opened the pull request:
Fixes #8112 

### What changes did you make? 
  -  Added Brandon Hu to the list of leaders

### Why did you make the changes (we will use this info to test)?
- updated the project information and made sure it was following correct order for website visitors to  find accurate information. 


<h3>CodeQL Alerts</h3>


After the PR has been submitted and the resulting GitHub actions/checks have been completed, developers should check the PR for CodeQL alert annotations.


<details><summary>Check the PR's comments. If present on your PR, the CodeQL alert looks similar as shown</summary>
  


</details>

Please let us know that you have checked for CodeQL alerts. **Please do not dismiss alerts.**
- [x] I have checked this PR for CodeQL alerts and none were found.
- [ ] I found CodeQL alert(s), and (select one):
   - [ ] I have resolved the CodeQL alert(s) as noted
   - [ ] I believe the CodeQL alert(s) is a false positive (Merge Team will evaluate)
   - [ ] I have followed the Instructions below, but I am still stuck (Merge Team will evaluate)

<details><summary>Instructions for resolving CodeQL alerts</summary>

If CodeQL alert/annotations appear, refer to [How to Resolve CodeQL alerts](https://github.com/hackforla/website/issues/6463#issuecomment-2002573270).  

In general, CodeQL alerts should be resolved prior to PR reviews and merging

</details>


<details>
<summary>Visuals before changes are applied </summary>

![image](https://github.com/user-attachments/assets/f4a55ed9-87bc-4600-ab63-a5a2e3203e33)

</details>
<details>
<summary>Visuals  after changes are applied</summary>

![image](https://github.com/user-attachments/assets/624cae28-e75c-4e01-91bf-e320cfa60463)

</details>

MohamedTwahir committed:
GFI added Brandon Hu8112

lastCoyotes reviewed (dismissed):
Changes made are to the requirements of the issue.
No other issues to report, so approving this PR!

Visual changes to the site are there, but the images provided are the files changed and not the site, I've included screenshots from the site and the build of this branch as an example:
<details>
<summary>Screenshots of the website. </summary>
Before:
<img width="1919" height="926" alt="image" src="https://github.com/user-attachments/assets/f16f19c4-3bd0-43f4-840e-706ccb329b43" />
After:
<img width="1919" height="927" alt="image" src="https://github.com/user-attachments/assets/b405cf95-0830-4bd9-95b0-bfd654811470" />
</details>

myronchen-git commented:
Review ETA: 8 PM 9/25/2025
Availability: Now

myronchen-git reviewed (changes requested):
### Done Correctly
- The merge from and to branches have been correctly set.
- The issue has been correctly linked.
- The pull request title is named properly.
- The explanation on what things were changed is good.
- The CodeQL alerts have been correctly checked off.
- Code changes have been correctly made.

### Required Changes
- In the pull request description, the text at the top, before the "Fixes" text, needs to be removed.
- In this pull request's **why** section, the reason should be related to the changes.  In this case, it's why Brandon Hu needs to be added.  Usually, you can derive the reason from the [issue's **Overview**](https://github.com/hackforla/website/issues/8112#issue-3036948134).  You can also review the [instructions for making a pull request](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#iii-complete-pull-request-3-explain-the-changes-you-made-then-explain-why-these-changes-were-needed) to get an idea of what to put.
- In this pull request's screenshot section, the screenshots are supposed to be visual changes that are not shown in the **Files changed** tab.  It should not be screenshots of code.  Please review [this part of the pull request instructions](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#iv-complete-pull-request-4-include-images-if-available), view the [example gif](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31b-complete-pull-request-on-hack-for-la-website-repo), and look at other pull requests on what kind of screenshots to use.
- In the [issue](https://github.com/hackforla/website/issues/8112), the last action item needs to be checked off.

### Optional Changes
- You can move Brandon Hu to immediately below Karen Coronel, so that the list is organized by role.  I don't remember if there is an official requirement for this, but it would really help visitors.

### Other Notes
- I noticed that you requested a review from **gmgonzal**.  If you did this through the right panel, and clicking on one of the suggestions, then it is not recommended that you do this.  The suggestions are drawn from all members on the website team, which has many inactive members.  A better way to get reviewers is to post a message on the website team's Slack channel or bringing this up in a meeting.

MohamedTwahir commented:
@lastCoyotes Thank you for your insightful comment. I will ensure that I follow the required guidelines optimally.

MohamedTwahir committed:
GFI  Added Brandon Hu #8343

MohamedTwahir commented:
@myronchen-git Thank you for your insightful comment i have managed to work on the changes needed let me know if all are okay.
Required Changes made 
1. In the pull request description - I have edited the description.
2. Updated  pull request's why section.
3. Added correct visuals.
4. Checked off the last action item.
<details>
<summary>Visuals before changes are applied and visual after changes are applied</summary>

<img width="835" height="649" alt="Before change" src="https://github.com/user-attachments/assets/42d42aa6-2317-47bc-be71-9dde894ddc3d" />


</details>
<details>
<summary>Visuals  after changes are applied</summary>

<img width="872" height="676" alt="After change" src="https://github.com/user-attachments/assets/624cae28-e75c-4e01-91bf-e320cfa60463" />

</details>

myronchen-git reviewed (changes requested):
Thank you for quickly making the changes.  Though, one thing was incorrectly done.

---

### Required Changes
- In this pull request's description's screenshot section, the screenshot before changes is supposed to be what the webpage looks like without *any* changes made.  It is not supposed to show Brandon Hu on the webpage yet.  Please attach a screenshot without Brandon Hu added.

myronchen-git reviewed (approved):
Well done.  Everything seems to be in order.
